src/Symfony/Bridge/Monolog/Tests/Processor/SwitchUserTokenProcessorTest.php

```diff
@@ -37,9 +37,10 @@ public function testProcessor()
 
         $expected = [
             'impersonator_token' => [
-                'username' => 'original_user',
                 'authenticated' => true,
                 'roles' => ['ROLE_SUPER_ADMIN'],
+                'user_identifier' => 'original_user',
```

Comment:
This requires bumping the security-core constraint to ^5.3 in the monolog bridge's composer.json.